src/plugins/htmlAudioPlayer/plugin.js

```diff
@@ -488,6 +492,33 @@ class HtmlAudioPlayer {
         return false;
     }
 
+    isAirPlayEnabled() {
+        if (document.AirPlayEnabled) {
+            return !!document.AirplayElement;
+        }
+        return false;
+    }
+
+    setAirPlayEnabled(isEnabled) {
+        const mediaElement = this._mediaElement;
+
+        if (document.AirPlayEnabled) {
+            if (mediaElement) {
+                if (isEnabled) {
+                    mediaElement.requestAirPlay().catch(function(err) {
+                        console.error('Error requesting AirPlay', err);
+                    });
+                } else {
+                    document.exitAirPLay().catch(function(err) {
+                        console.error('Error exiting AirPlay', err);
+                    });
+                }
+            }
+        } else {
+            mediaElement.webkitShowPlaybackTargetPicker();
+        }
```

Comment:
I know this is copied from `htmlVidePlayer`, but I believe (_not tested_) the conditions should look like
```suggestion
        if (mediaElement) {
            if (document.AirPlayEnabled) {
                if (isEnabled) {
                    mediaElement.requestAirPlay().catch(function(err) {
                        console.error('Error requesting AirPlay', err);
                    });
                } else {
                    document.exitAirPLay().catch(function(err) {
                        console.error('Error exiting AirPlay', err);
                    });
                }
            } else {
                mediaElement.webkitShowPlaybackTargetPicker();
            }
        }
```
Otherwise, in the else block, the `mediaElement` is accessed without checking.

Reply:
Yes, it was copied from HtmlVideoPlayer with a few changes I follow the writing style in Htmlvideplayer. 

About your suggestion, the airplay button in Safari Desktop is inactive if I use the condition. So your suggestion does not work.

Reply:
Weird, it's just a rearrangement of conditions.

The current structure is wrong:
```js
if (AirPlayEnabled) {
    if (mediaElement) {
        mediaElement...
    }
} else {
    // Access without check
    mediaElement...
}
```

Proposed structure:
```js
if (mediaElement) {
    if (AirPlayEnabled) {
        mediaElement...
    } else {
        // Access with check
        mediaElement...
    }
}
```

In fact, checking `mediaElement` doesn't seem necessary (when UI appears, the media element already exists), but other functions have a check.

Reply:
@stamatovg could you take another look at the comment above? Thanks!